middle-ram.cpp

```diff
@@ -38,6 +38,11 @@ void middle_ram_t::nodes_set(osmium::Node const &node)
     cache->set(node.id(), node.location());
 }
 
+osmium::Location middle_ram_t::nodes_get(osmid_t osm_id)
+{
+    return cache->get(osm_id);
+}
+
```

Comment:
There are there are also other locations where nodes_get would have to be implemented, see compiler errors:

```
In file included from /home/travis/build/openstreetmap/osm2pgsql/tests/test-parse-extra-args.cpp:15:

/home/travis/build/openstreetmap/osm2pgsql/tests/mockups.hpp:7:8: note:   because the following virtual functions are pure within ‘dummy_middle_t’:

 struct dummy_middle_t : public middle_t {

        ^~~~~~~~~~~~~~

In file included from /home/travis/build/openstreetmap/osm2pgsql/tests/test-parse-extra-args.cpp:9:

/home/travis/build/openstreetmap/osm2pgsql/middle.hpp:91:30: note: 	‘virtual osmium::Location middle_t::nodes_get(osmid_t)’

     virtual osmium::Location nodes_get(osmid_t osm_id) = 0;
```